Src/Clients/WebUI/Controllers/Storage/ActorsWebUiController.cs

```diff
@@ -16,30 +18,55 @@ public class ActorsWebUiController : BaseController
     {
         private readonly IApiTools _apiTools;
         private readonly IAuthorizeApiTools _authorizeApiTools;
+        private readonly ILogger<ActorsWebUiController> _logger;
 
-        public ActorsWebUiController(IApiTools apiTools, IAuthorizeApiTools authorizeApiTools)
+        public ActorsWebUiController(IApiTools apiTools, IAuthorizeApiTools authorizeApiTools,
+            ILogger<ActorsWebUiController> logger)
         {
             _apiTools = apiTools;
             _authorizeApiTools = authorizeApiTools;
+            _logger = logger;
         }
 
         [HttpGet]
-        public IActionResult Index()
+        public async Task<IActionResult> Index()
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Add the 'Async' suffix to the name of this method.](https://app.codacy.com/manual/liannoi/team-project/pullRequest?prid=5600534)